### PR TITLE
testing/getmail: update to 5.13

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: testing/getmail/APKBUILD [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
-depends_dev=""
 makedepends="python2-dev"
-install=""
 subpackages="$pkgname-doc"
 options="!check" # contains no tests
 source="http://pyropus.ca/software/getmail/old-versions/$pkgname-$pkgver.tar.gz"
 builddir="$srcdir"/$pkgname-$pkgver
 
 build() {
-	cd "$builddir"
```
Comment (andypost) [inline testing/getmail/APKBUILD]: No reason to remove

File: testing/getmail/APKBUILD [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
 	python2 setup.py build
 }
 
 package() {
-	cd "$builddir"
```
Comment (andypost) [inline testing/getmail/APKBUILD]: @ncopa asked to keep it for a while

**Comment by maxice8** [comment]
Reverted

File: testing/getmail/APKBUILD (hunk 1 above)
Comment (Ikke) [reply testing/getmail/APKBUILD]: @andypost I've kinda been telling that `cd "$builddir"` is redundant nowadays..

Comment (maxice8) [reply testing/getmail/APKBUILD]: ncopa asked to keep them at least in main (https://github.com/alpinelinux/aports/pull/7244)

If you want i can expand the keeping 'cd "$builddir"' to all repos.

**Pull request closed without merging**